app/src/marketplace/containers/ProductPage/PurchaseModal.jsx

```diff
@@ -92,6 +92,8 @@ export const PurchaseDialog = ({ productId, api }: Props) => {
     }, [dispatch, loadEthIdentities, loadContractProduct, productId, isMounted])
 
     useEffect(() => {
+        if (!account) { return }
```

Comment:
Added this to prevent a console warning when opening purchase modal, no need to fetch balances before Metamask is unlocked (ie. account is not null)